### PR TITLE
Updated install videos and filesystem paths article

### LINKED ISSUE #2706: Redo Zui Installation videos & update Filesystem Paths doc
The videos at https://zui.brimdata.io/docs/Installation are still mostly ok, but need some updates to reflect the new "Zui" name and also show the "per-machine" install option that was added in #2686. On that latter point, the [Filesystem Paths](https://zui.brimdata.io/docs/support/Filesystem-Paths) article also needs an update to reflect the different pathnames on Windows the user may have based on whether they're using a per-user or per-machine install. The videos should also start using the newer volcano graphic.

### PR DESCRIPTION
**Pull request by philrz**
Now that Zui v1.0.0 is out, the animated GIFs that have been used for install guidance are due for an update, which is covered in this PR. I've also expanded the Filesystem Paths article with the separate Windows guidance for the single-user & all-user install types. Finally, I've dropped the link to the old Brim demo video since Zui is now different enough that I expect a new user would be confused by it if looking at it side-by-side next to current Zui. I hope to soon create some new entry-level Zui demo videos that could take its place.

While video editing is potentially time consuming, since these videos are short and I have the originals in Camtasia, making edits would be pretty easy. Reviews should not be shy if you have ideas for improvement.

So you can easily see the animated GIFs and the rendered docs pages in their updated forms, I've done a deployment of what's in this branch to a staging site in a personal Netlify at https://jocular-choux-3cf6d8.netlify.app/docs.

Closes #2706

### CODE REVIEW

File: docs/Installation.md [hunk 1]
```diff
@@ -28,19 +28,16 @@ updating to a newer version of the app will preserve your existing
...
 and we'll be happy to help.
 
 ## Windows Installation
 
-* Download the Zui installer via the Windows link at the [Zui download](https://www.brimdata.io/download/) page
+* Download the Zui installer via the **Windows** link at the [Zui download](https://www.brimdata.io/download/) page
 * Launch the downloaded `.exe` file to begin installation
-* When prompted, click "I Agree" to accept the terms of the license agreement
-* Zui will start automatically when install completes
+* Choose between install for only the current user (default) or a machine-wide install
```
Comment (nwt) [inline docs/Installation.md:39]:
```suggestion
* Choose between installing for only the current user (default) or a machine-wide install
```

File: docs/Installation.md [hunk 2]
```diff
@@ -28,19 +28,16 @@ updating to a newer version of the app will preserve your existing
...
+* Click **Finish** and Zui will launch immediately
```
Comment (nwt) [inline docs/Installation.md:40]: This often isn't immediate for me.
```suggestion
* Click **Finish** and Zui will launch when installation completes
```

File: docs/support/Filesystem-Paths.md [hunk 3]
```diff
@@ -13,11 +13,12 @@ data is stored. Below you'll find these details for each supported platform.
...
 application are unpacked to the following path on each platform:
 
-|**OS Platform**|**Location**                                 |
-|---------------|---------------------------------------------|
-| **Windows**   | `%USERPROFILE%\AppData\Local\Programs\Zui`  |
-| **macOS**     | `/Applications/Zui.app`                     |
-| **Linux**     | `/opt/Zui`                                  |
+|**OS Platform**|**Install Type**|**Location**                                |
+|---------------|----------------|--------------------------------------------|
+| **Windows**   | Single-user    | `%USERPROFILE%\AppData\Local\Programs\Zui` |
```
Comment (nwt) [inline docs/support/Filesystem-Paths.md:18]:
```suggestion
| **Windows**   | Current user    | `%USERPROFILE%\AppData\Local\Programs\Zui` |
```

File: docs/support/Filesystem-Paths.md [hunk 4]
```diff
@@ -30,11 +31,12 @@ bundled with the app, such as [`zq`](https://zed.brimdata.io/docs/commands/zq/),
...
 found in the `zdeps` subdirectory as follows:
 
-|**OS Platform**|**Location**|
-|---------------|------------|
-| **Windows**   | `%USERPROFILE%\AppData\Local\Programs\Zui\resources\app.asar.unpacked\zdeps` |
-| **macOS**     | `/Applications/Zui.app/Contents/Resources/app.asar.unpacked/zdeps` |
-| **Linux**     | `/opt/Zui/resources/app.asar.unpacked/zdeps` |
+|**OS Platform**|**Install Type**|**Location**                                                                  |
+|---------------|----------------|------------------------------------------------------------------------------|
+| **Windows**   | Single-user    | `%USERPROFILE%\AppData\Local\Programs\Zui\resources\app.asar.unpacked\zdeps` |
```
Comment (nwt) [inline docs/support/Filesystem-Paths.md:36]:
```suggestion
| **Windows**   | Current user    | `%USERPROFILE%\AppData\Local\Programs\Zui\resources\app.asar.unpacked\zdeps` |
```

**Review by jameskerr — COMMENTED** [review]
This looks great Phil. Thank you!

**Review by nwt — APPROVED** [review]
:shipit:

**Pull request merged**